cumulusci/robotframework/utils.py

```diff
@@ -229,3 +232,31 @@ def wrapper(*args, **kwargs):
             raise
 
     return wrapper
+
+
+def get_locator_module_name(version):
+    """Return module name of locator file for the specified version
+
+    If a file for the specified version can't be found, the locator
+    file with the highest number will be returned. Passing in None
+    effectively means "give me the latest version".
+    """
+
+    here = os.path.dirname(__file__)
+    if os.path.exists(os.path.join(here, f"locators_{version}.py")):
+        # our work here is done.
+        locator_filename = f"locators_{version}.py"
+    else:
+
+        def by_num(filename):
+            """Pull out the number from a filename"""
+            nums = re.findall(r"_(\d+)", filename)
+            if nums:
+                return int(nums[-1])
+            return 0
+
+        files = sorted(glob.glob1(here, "locators_*.py"), key=by_num)
```

Comment:
`glob1()` is [not documented](https://docs.python.org/3/library/glob.html) in the current standard library and I believe is considered private. Can we use `Pathlib`'s glob instead (https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob)?

Reply:
Hmmm. I don't recall why I switched to glob1. Good catch. I'll use the code from pathlib.